test/tests/connection.js

```diff
@@ -0,0 +1,22 @@
+let request = require('../request');
```

Comment:
So change the `tests` directory to `routes`? I'm fine with that, although will probably need to rename a few files so they make sense with that convention.